WorkerApi/ArmoniK.DevelopmentKit.WorkerApi/ServiceRequestContext.cs

```diff
@@ -122,16 +125,21 @@ public byte[] Execute(ITaskHandler taskHandler)
 
 public class ServiceRequestContext
 {
+  [CanBeNull]
```

Comment:
Don't we have `nullable` enabled on the project?

Reply:
not yet. it's part of refactoring